eval/eval_chunking_grading.py

```
@@ -117,6 +586,47 @@ def load_rubric(path: Path) -> str:
 
+def load_rubric_from_excel(excel_path: Path, sheet_name: str = "Question Details", question_label: str | None = None) -> str:
+    """
+    Build a rubric string from an Excel sheet (e.g. 24fallmetcs581_m1 Quiz 1.xlsx, 'Question Details' tab).
+    If question_label is provided (e.g. 'Question 13'), we try to extract only rows mentioning that;
+    otherwise we join all non-empty rows.
+    """
+    import pandas as pd
+
+    try:
+        df = pd.read_excel(excel_path, sheet_name=sheet_name)
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)